source/accessibility.html.md.erb

```diff
@@ -17,49 +17,49 @@ This website is run by the GOV.UK Account team at the Government Digital Service
 + navigate most of the website using speech recognition software
 + listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
 
-We’ve also made the website text as simple as possible to understand.
+We've also written the website in plain English to make it as simple as possible to understand.
```

Comment:
do we want to use the `’` characters for apostrophes?

Reply:
I've always used this character in tech content and it hasn't been a problem before